### ISSUE TITLE
Shadowy Assassins in Southshore

### ISSUE THREAD

**Issue by Avelorian** [Wall-core/Everlook-Bugtracker#313]
Guards around Southshore dont agro on assasins, which is incorrect, so low lvl players just dying from 37-38 lvl mobs, who chase forever.

**Comment by NickTyrer** [comment]
Closed here https://github.com/vmangos/core/pull/1630

**Comment by CarkMcBark** [comment]
yeah... this is huge problem... and there is A LOT of them aswell. just killing you while guards stands there like a tree good for nothing hahaha
